test/augmentation/test_perspective_rand.py

```diff
@@ -42,6 +42,17 @@ def test_smoke_transform(self, device):
         assert aug.transform_matrix.shape == torch.Size([1, 3, 3])
         assert aug.inverse(out_perspective).shape == x_data.shape
 
+    def test_smoke_transform_sampling_method(self, device):
```

Comment:
```suggestion
    def test_smoke_transform_sampling_method(self, device, dtype):
```
@shijianjian @twsl can be done later, but is there any reason why don't test different data dtypes ?

Reply:
@Parskatt Can you add dtype to all the tests here?

Reply:
@shijianjian sure, should be fixed now.